.editorconfig

```diff
@@ -1,20 +1,477 @@
-# http://editorconfig.org
-
 root = true
 
 [*]
-indent_style = space
-indent_size = 4
-trim_trailing_whitespace = true
-insert_final_newline = true
 charset = utf-8
 end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = false
+max_line_length = 88
+tab_width = 4
+ij_continuation_indent_size = 8
+ij_formatter_off_tag = @formatter:off
+ij_formatter_on_tag = @formatter:on
+ij_formatter_tags_enabled = false
+ij_smart_tabs = false
+ij_wrap_on_typing = false
```

Comment:
Properties that are specific to Idea seems to be the opposite of what .editorconfig should do.
There should not be properties that are IDE specific.

Reply:
done